shell/src/test/java/org/apache/accumulo/shell/ShellSetInstanceTest.java

```diff
@@ -119,17 +120,15 @@ public void tearDown() {
     SiteConfiguration.clearInstance();
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
@ctubbsii what do you think of this for a test like this thats testing something related to mock?